Dynatrace/dynatrace-operator
# Adds ActiveGate endpoints as Istio ServiceEntries

## Description

- adds ActiveGate endpoints as well as OneAgent communication hosts as Istio service entries

## How can this be tested?

- install istio
- deploy dynakube
- check serviceentries: should contain oneagent and activegate endpoints

## Checklist
